yuca/wrappers/glider_wrapper.py

```diff
@@ -138,6 +148,9 @@ def init_displacement_grid(self):
         self.x_grid = torch.tensor(xx).reshape(1, 1, self.dim, self.dim)
         self.y_grid = torch.tensor(yy).reshape(1, 1, self.dim, self.dim)
 
+        self.x_grid = self.x_grid / self.ca.kernel_radius
+        self.y_grid = self.y_grid / self.ca.kernel_radius
```

Comment:
Mobiity is normalized w.r.t. the kernel radius, _i.e._ the maximum speed for information to travel in the simulation (aka the speed of light)